source/Calamari/Commands/DownloadPackageCommand.cs

```diff
@@ -62,10 +62,10 @@ public class DownloadPackageCommand : Command
                     }
                     versionFormat = format;
                 });
-            Options.Add("feedId=", "Id of the NuGet feed", v => feedId = v);
```

Comment:
This change is purely aesthetic - the CLI interface is not specific (any more) to NuGet feeds.